File: modul-02/python/PRAK202-2310817310003-Mfaridsetiawan.py
a = float(input("Masukkan Nilai Pertama :"))
b = float(input("Masukkan Nilai Kedua   :"))

hasil = a + b

a = ( f"{int(a)}" if a.is_integer() else f"{a:.2f}")
print(f'Hasil dari penjumlahan nilai a "{a}" dan nilai kedua "{b}" adalah "{hasil:.2f}"')

File: modul-02/bahasa c/PRAK202-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

int main(){
    float a;
    float b;
    printf("Masukkan Nilai Pertama  :");
    scanf("%f", &a);
    printf("Masukkan Nilai Kedua    :");
    scanf("%f", &b);
    float hasil = a+b;
    printf("Hasil dari penjumlahan nilai pertama \"%g\" dan nilai kedua \"%g\" adalah \"%.2f\"", a , b, hasil);
   
    return 0;

}